bugzilla2gitlab/models.py

```diff
@@ -262,17 +268,29 @@ def parse_filename(cls, comment):
     def save(self):
         url = "{}/attachment.cgi?id={}".format(conf.bugzilla_base_url, self.id)
         result = _perform_request(url, "get", json=False)
+        filename = self.file_description
+        # Use real filename to store attachment but descriptive name for issue text
+        if 'Content-disposition' in result.headers:
```

Comment:
I think if/else is more clear in this situation than setting the default value followed by an if statement that likely changes it. Also, wouldn't it be an error if no filename can be parsed from the `Content-disposition`?

Reply:
Please review the refactored version. I agree that not being able to parse `Content-disposition` is an exception so I made it an Exception.